docs/self-managed/setup/deploy/amazon/amazon-eks/eks-helm.md

```diff
@@ -42,10 +54,44 @@ To streamline the execution of the subsequent commands, it is recommended to exp
 
 The following are the required environment variables with some example values:
 
+```bash reference
+https://github.com/camunda/camunda-tf-eks-module/blob/feature/opensearch-doc/examples/camunda-8.6/procedure/chart-env.sh
+```
+
+<Tabs groupId="env">
+
+  <TabItem value="standard" label="Standard" default>
+  
+When using Basic authentication (username and password), some following environment variables must be set and contain valid values.
```

Comment:
or maybe some instead of following.
Not entirely sure.

Maybe rephrase to

```
When using Basic authentication (username and password), specific environment variables need to be set with valid values. After setting these variables, you can confirm they are configured correctly by running the following loop:
```